Clarify that MAC is supported for VCs and presentations (#230)
oauth-wg/oauth-sd-jwt-vc: draft-ietf-oauth-sd-jwt-vc.md

In the so-called Issuer-Holder-Verifier Model, Issuers issue so-called Verifiable Credentials to a
Holder, who can then present the Verifiable Credentials to Verifiers. Verifiable

> Credentials are cryptographically secured statements about a Subject, typically the Holder. A Holder can present these statements to a Verifier in a repudiable or non-repudiable way.

Reviewer: I'm not sure if we can just drop those words here and expect people to understand what this means.

> Credentials are cryptographically secured statements about a Subject, typically the Holder. A Holder can present these statements to a Verifier in a repudiable or non-repudiable way.

Author: ```suggestion
Credentials are cryptographically secured statements about a Subject, typically the Holder.
```

[@!I-D.ietf-oauth-selective-disclosure-jwt].

Verifiable Credential (VC):

> :  An assertion with claims about a Subject that is cryptographically secured by an Issuer, which a Verifier can authenticate in a repudiable or non-repudiable manner. Repudiable assertions can employ MAC-based securing mechanisms, provided that the Issuer of the assertion can be confirmed.

Reviewer: As above, I'm not sure the first sentence works without further explanation.

> :  An assertion with claims about a Subject that is cryptographically secured by an Issuer, which a Verifier can authenticate in a repudiable or non-repudiable manner. Repudiable assertions can employ MAC-based securing mechanisms, provided that the Issuer of the assertion can be confirmed.

Author: ```suggestion
:  An assertion with claims about a Subject that is cryptographically secured by an Issuer (usually by a digital signature).
```
